BedrockServer.cpp

```diff
@@ -852,18 +852,20 @@ void BedrockServer::worker(int threadId)
             // the `_futureCommitCommands` queue.
             if (state == SQLiteNode::FOLLOWING && command->escalateImmediately && !command->complete) {
                 if (_escalateOverHTTP) {
-                    if (_clusterMessenger.sendToLeader(*command)) {
-                        SINFO("Immediately escalating " << command->request.methodLine << " to leader.");
-                        waitForHTTPS(move(command));
+                    SINFO("Beginning immediately escalating " << command->request.methodLine << " to leader.");
```

Comment:
NAB: this log line is going to happen a fair bit and doesn't add a ton of value I can see, since the "runOnLeader" either completes and gets logged or fails and gets logged.

Reply:
There's a `TODO` to pull out the excessive logging once we've tested this, though it's in `SQLiteClusterMessenger`.

Reply:
Ah, I thought that was referring to that one specific line.